docpages/02_build.md

```diff
@@ -103,16 +101,23 @@ For voice support, additional dependencies are required:
 
 ## 3. Build Source Code
 
-    mkdir build
-    cd build
-    cmake ..
-    make -j8
+    cmake -B ./build
+    cmake --build ./build -j8
     
 Replace the number after -j with a number suitable for your setup, usually the same as the number of cores on your machine. `cmake` will fetch any dependencies that are required for you and ensure they are compiled alongside the library.
 
 ## 4. Optional: Run test cases
 
-Run `./test` for unit test cases. You will need to create a `config.json` file in the directory above the executable file with a valid bot token in it. See the example file `config.example.json` for an example of the correct format.
+Run `./test` for unit test cases. 
```

Comment:
running unit tests is not really encouraged, this should probably be its own page under the advanced section, as only developers/contributors to the library need to do this step to verify their PR hasnt broken anything. The new page should probably also detail how to add new unit tests to test.cpp.

Reply:
I'll remove the testing part from here, and I'll create a separate PR for testing.